noir/noir-repo/acvm-repo/acir/src/circuit/opcodes.rs

```diff
@@ -30,6 +30,8 @@ pub enum Opcode {
     MemoryInit {
         block_id: BlockId,
         init: Vec<Witness>,
+        /// 0 for regular memory, 1 for call-data and 2 for return-data
+        databus: u8,
```

Comment:
I feel an enum would be clearer here and it also aligns with how we have been distinguishing opcodes already

Reply:
Done